api/snippets-menu/aou-snippets-menu.js.template

```diff
@@ -33,6 +33,12 @@ define([
       // Avoid mutating the original default_menus to reduce side-effects.
       snippets_menu.options['menus'] = [...snippets_menu.default_menus];
       const menus = snippets_menu.options['menus'];
+
+      // Remove pandas because it is broken by https://github.com/ipython-contrib/jupyter_contrib_nbextensions/pull/1636
```

Comment:
looks like this change was included by mistake

Reply:
ah. Good catch